src/main/java/net/greatstart/controllers/UserController.java

```diff
@@ -1,39 +1,60 @@
 package net.greatstart.controllers;
 
+import net.greatstart.dto.DtoUser;
 import net.greatstart.model.User;
+import net.greatstart.services.UserConverterService;
 import net.greatstart.services.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
-
-import java.util.List;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
 
 @Controller
+@RequestMapping("/user")
 public class UserController {
 
     private UserService userService;
+    private UserConverterService userConverter;
 
     @Autowired
-    public UserController(UserService userService) {
+    public UserController(UserService userService, UserConverterService userConverterService) {
         this.userService = userService;
+        this.userConverter = userConverterService;
     }
 
-    public boolean createUser(User user) {
-        return userService.createUser(user);
-    }
-
-    public boolean updateUser(long id) {
-        return userService.updateUser(id);
+    // TODO error pages (no such user etc.)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)